### PR TITLE
Tweaked some small human ships

### PR DESCRIPTION
**Pull request by Nescio0**
Summary:
* Hawk  +10 weapon capacity, –10 cargo space
* Wasp +2 weapon capacity
* Bounder –5 weapon capacity
* Headhunter  +10 weapon capacity, –10 cargo space
* Barb and Barb (Proton) –2 engine capacity
* Heavy Shuttle +100 shields
* Shuttle +100 shields, –100 hull
* Arrow –0.5 drag

Currently the Fury and the Hawk both have a weapon capacity of 40, allowing them to be equipped with two heavy rocket launchers or one typhoon launcher. However, the Hawk is a clearly larger ship and it's desciption states it's "much more heavily armed". Therefore I think it deserves more weapon capacity; this proposal increases it to <s>45</s> 50, which allows the Hawk to use two plasma cannons or torpedo launchers.
Furthermore, most interceptors have a cargo space of 10, the Fury has 15, but the Hawk has 30, equal to the Raven, a much larger ship. Therefore I propose to reduce the Hawk's from 30 to 20, which seems more reasonable (and is still enough to steal a jump drive).

The Wasp's weapon capacity is increased from 28 to 30, making it more versatile and less useless, and allowing it to compete with the Berserker (35), so both can equip three meteor launchers. This would put the Wasp, "a medium-strength interceptor", in between the Sparrow (can equip two meteors) and Fury (can equip four meteors).

The Bounder, explicitly "designed not for battle", currently has a weapon capacity of 50. This is reduced to 45, more than enough for its default build (30) or its variant (44).
<s>Furthermore, its current engine capacity (110) is very close to much larger ships, such as the Leviathan (120) or Star Queen (100), and far more than its default and variant builds need (81). The engine capacity is reduced to 90, still higher than e.g. the Arrow (60) or Heavy Shuttle (60).</s>

The Headhunter currently has a weapon capacity of 60, the same as the Quicksilver, which has no turret mount; by increasing it to 70, the Headhunter could equip a heavy laser turret and two heavy lasers, or an anti-missile turret and two torpedo launchers. Besides, it would fit nicely in between the Scout (40) and Hawk (50) on the one hand, Raven (90) and Gunboat (100) on the other.
Furthermore, reduced the Headhunter's cargo space from 50 to 40, for the same reasons as the Hawk's.

The Barb's description states it's "featuring newly-developed miniaturized ion engines", i.e. the `X1050 Ion Engines`, the description of which starts with "When designing the Barb". However, the X1050's size is 20, therefore the Barb doesn't need 22 engine capacity.

According to its description, the Arrow is "able to outrun nearly every other ship in the sky." Although its maximum speed (c. 600 per second) is above average, it is not really exceptional. Besides, it can't be sped up by installing more powerful engines, because it already has the largest atomic engines its capacity allows. By reducing its drag, its maximum speed is increased by c. 20%, allowing it to at least outrun most default human ship builds.

<s>Also changed Flivver category to "Interceptor", because its cargo size and bunks are very low.</s>

### CODE REVIEW

**Comment by Bladewood** [comment]
Flivver's a transport in the same way that a gascraft or an arrow is.

I really think the smallest human ships are not meant to handle a pair of some of the biggest and strongest human weapons: two electrons, which are only slightly smaller than plasma cannons; and plasma turrets are considered *huge*.

**Comment by DingusShingleton** [comment]
A taxi has the same amount of passenger slots as a tesla model s, but they are VERY different types of automobile

**Comment by Nescio0** [comment]
To clarify, this proposal does *not* change any outfits, nor does it add gun ports (#3350 does). All this one does is increase Hawk's weapon capacity from 40 to 45 and Wasp's from 28 to 30, a small tweak that won't fundamentally change the game.
As for the flivver, to me it looks (visually and statistically) rather similar to the sparrow; an arrow is quite different.

**Comment by Amazinite** [comment]
Ref https://github.com/endless-sky/endless-sky/pull/3350#issuecomment-350156778 and the comments that refer to it in terms of the Hawk change. 

I'm averse to the Wasp change, as the Wasp is explicitly stated as being designed to carry a Proton Gun, and what do you know, it currently has exactly the amount of weapon space required to do that. The addition of two weapon space just so it can equip three meteor missiles doesn't seem fitting to me.

If we're going to argue statistical and visual similarities, then this is just going to stumble into the same arguments as when we discussed changing the Modified Argosy's category from light warship to medium warship.

Could you actually give a reason for your changes in the OP instead of just listing the changes that you did (which we can see in the files changed anyways), and instead of listing your reasoning in replies after we ask?

**Comment by Nescio0** [comment]
That's the Barb (Proton); the Wasp's description reads: 
	description "The wasp is a medium-strength interceptor, designed by Syndicated Shipyards to serve as an escort for cargo vessels. Like most Syndicate ships, it is not particularly pretty, but it at least serves the purpose it was made for."

By giving it 30 weapon capacity instead of 28, the Wasp becomes more versatile, allowing it to compete with the Berserker.

**Comment by Nescio0** [comment]
45 is the next step up from the previous interceptor. Besides, I prefer different ships to have different capacities; currently there is no human ship with a weapon capacity of 45, although two (Bounder and Scout) have 50
However, if you think 50 is a better value for the Hawk than 45 or 40, then I'll happily change it.

**Comment by Amazinite** [comment]
* **Hawk**: fine by me. I see a benefit in buffing the Hawk's weapon space. Given its other stats, the Hawk is clearly intended to be the heaviest of the human interceptors. Its odd then that its current weapon space matches that of the Fury. 50 weapon space allows the Hawk to dual plasma cannons or torpedoes (perhaps not effectively, though), making it an interesting choice as a transition between interceptors and light warships. I've tested a 50 weapon space Hawk with dual plasma cannons in the past and the cooling required means you need to sacrifice engine space, so I don't see any problems arising there. Dual torpedoes allows for an interesting glass cannon loadout that allows the Hawk to punch up, and dual electron beams are also a completely viable option to use on a Hawk (I was able to make a highly sustainable loadout that didn't require me to sacrifice anything major, aside from perhaps stronger shield regen), but as I mentioned previously, I don't see this as being a problem since by the time a player has access to electron beams, they also have access to ships much better than an interceptor. No real objection to the cargo change.
* **Wasp**: I'm not entirely convinced of the reasoning for this change. The Beserker is a clearly better interceptor, being better than the Wasp in nearly every way, therefore I don't see the Wasp as needing a weapon space buff in order to try and make it compete with the Beserker. And while I was wrong on the Wasp being explicitly stated as being designed to carry a Proton Gun, I think it's at least fair to say that it is implicitly stated in the fact that the Wasp has exactly enough space to carry a Proton Gun. I feel that leaving the Wasp at 28 weapon space would keep it more fitting as a Syndicate ship, being able to do what it was designed for and nothing more.
* **Bounder**: nerfs are a whole different ballgame from buffs, and I'm not entirely convinced here. I could live with the slight weapon space nerf to the Bounder, even if I don't find it necessary, but I'm going to have to say no to the engine space nerf, especially given how considerable it is. For starters, I don't think that the size of the Bounder is a good grounds for an engine space nerf, especially when comparing it to ships in a different category (comparing it to the Leviathan). The Scout has 100 engine space, and it is a very small ship, and the Blackbird, which is comparable in size to the Bounder, also has 110 engine space. It's also important to note that the Star Queen as 100, not 120 engine space. And just because the Bounder could be nerfed in its engine space without needing to change how it is outfitted in game doesn't mean it should; that 20 engine space could very easily be utilized by the player, as it is important to note that almost no ships in the game have optimal loadouts that utilize all their space.
* **Headhunter**: fine by me. The Headhunter has always felt like an odd ship to me given its similarity to the Quicksilver with 60 weapon space but with the addition of a turret. I've always felt like I either have to fill my hardpoints with weak weapons or leave the turret a wasted hardpoint with nothing in it so that I can at least have powerful guns. 

TL;DR Hawk and Headhunter changes are fine, not convinced on the Wasp changes but wouldn't fight over it, disagree with the Bounder changes, at least in the engine space department.

**Comment by Janaszar** [comment]
@Amazinite Brought this up in the Discord, where I posted my original response: (edited for grammar, since I wrote it as a response to Amazinite, but now it's to you)

Personally, never bother too much with interceptors since they die so easily late-game, and early-game for me tends to focus on (heavy) lasers for major weapon choice.

Hawk: I can see where you're going with this.  Hawk is a bit of the "odd one out" with the extra cargo space, but the extra weapon space makes it pretty much the "best" interceptor then (aside from Marauder versions).  There's nothing wrong with the extra cargo space; I think of it like being along the lines of the Behemoth idea of a "moderately" armed transport vessel.

Wasp: I love the design, but it's just so useless to me... Honestly, +2 isn't a big deal; if it was written up to explicitly take advantage of the Proton Gun I could see why it should stay, but I see no reason to say no right now.

Bounder: Weapon Capacity, sure.  But Engines?  Nope, nope, nope.  While Stock uses Ions, it can fit an A370/A375 with 19 to spare.  Which, if we ever had a "high performance" variant, I would expect it to use those and maybe an A125 or an Afterburner to give it some great Acceleration and Turning.  It can also fit an Impala Plasma set with 9 to spare.

Headhunter: I've always felt the Headhunter was like an "Interceptor+" sort of ship; slightly better than the best (Normal Human) interceptor to push it into the Light Warship category, but not really a true "Light Warship".  I can go either way on this, as it seems to me more like a "well, you have to pick either to use a turret, or a pair of guns." situation than a "you should have weapons in every slot" one.

**Comment by Nescio0** [comment]
Thank you for your constructive feedback! I've now reverted the bounder engine nerf.

**Comment by Nescio0** [comment]
Thank you for your constructive feedback! I've now reverted the bounder engine nerf.
As for the Wasp's +2, a weapon capacity of 28 is enough for three flamethrowers, two sidewinders, or one proton gun. 30 would allow three meteors or one particle cannon. The Berserker's 35 allows four gatling guns or one typhoon launcher.

**Comment by Amazinite** [comment]
Ref capitalization changes to Navy ships: #2698
I'd say save it for another PR since the stated purpose of this PR is for stat changes.

**Comment by Nescio0** [comment]
As for the capitalization, these ship descriptions deal with the specific ship, not a generic category. E.g. the Kar Ik Vot 349 is a combat drone, but that's irrelevant in the description of the Combat Drone (Navy ship).

As for the Wasp, let's try to put it differently: at 30 weapon capacity the Wasp (three meteors) would be clearly better than the Sparrow (two meteors) and clearly worse than the Fury (four meteors). +2 is not a large tweak, but it does make the Wasp less useless.

**Comment by Amazinite** [comment]
>As for the Wasp, let's try to put it differently: at 30 weapon capacity the Wasp (three meteors) would be clearly better than the Sparrow (two meteors) and clearly worse than the Fury (four meteors). +2 is not a large tweak, but it does make the Wasp less useless.

That's only looking at meteor missiles, though. I could make the same comparison by saying that the Wasp (three beam lasers or gatling guns) is clearly better than the Sparrow (two beam lasers or gatlings) and worse than the Fury (four beam lasers or gatlings). I'm just not sure what the fixation on meteor missiles is for. We also can't only look at one weapon at a time. For example, one could look at it like this:


Ship | What it could hold 
----- | -----------
Sparrow | 2 Meteor Missiles
Wasp | 2 Meteor Missiles + 1 Gatling Gun/Beam Laser
Berserker | 3 Meteor Missiles + 1 Energy Blaster
Fury | 4 Meteor Missiles
Hawk | 2 Torpedoes (with this PR)

A progression already exists among the interceptors if you're only wanting to look at meteor missiles, but the Wasp clearly already holds an intermediate spot between the Sparrow and the Berserker if you look at mixing weapons instead of only trying to look at one weapon

**Comment by Nescio0** [comment]
> I'm just not sure what the fixation on meteor missiles is for. 
The meteor launcher is just a convenient choice, because of its round size (10), and because when you want to maximize the damage of your interceptor, you'd probably choose meteors.
We could also look at e.g. javelins:
Sparrow: one javelin and one blaster (17/20)
Flivver: two javelins (24/25)
Wasp (currently): two javelins (24/28)
Wasp (proposed): two javelins and one blaster (29/30)
Berserker: two javelins and two blasters (34/35)
Fury: three javelins (36/40)

By giving the Wasp a tiny +2 weapon capacity, it suddenly has many more weapon outfit possibilities, making it more versatile and less useless. I highly doubt anyone ever purchase a Wasp at the moment, and if someone might purchase it because it can equip three meteor, great, that's one entirely useless ship promoted to a mostly useless one.

**Comment by Nescio0** [comment]
> I'm just not sure what the fixation on meteor missiles is for. 

The meteor launcher is just a convenient choice, because of its round size (10), and because when you want to maximize the damage of your interceptor, you'd probably choose meteors.
We could also look at different weapons, of course, e.g. javelins:
Sparrow: one javelin and one blaster (17/20)
Flivver: two javelins (24/25)
Wasp (currently): two javelins (24/28)
Wasp (proposed): two javelins and one blaster (29/30)
Berserker: two javelins and two blasters (34/35)
Fury: three javelins (36/40)

By giving the Wasp a tiny +2 weapon capacity, it suddenly has many more weapon outfit possibilities, making it more versatile and less useless. I highly doubt anyone would ever purchase a Wasp at the moment, but if someone might buy it because it can equip three meteors, great, then it's mostly useless instead of always entirely useless.
> "The Wasp is a medium-strength interceptor"

**Comment by Amazinite** [comment]
Agree to disagree at this point. I feel that the Wasp is already in a perfect position with its weapon space, and while +2 weapon space does increase the customizability of the Wasp (but where can't that be said when increasing a ship's outfit, weapon, or engine space?), I feel that it harms the current feel of the Wasp as being a Syndicate ship designed to perfectly fit a Syndicate weapon, the Proton Gun. Looking at it from a design perspective, why would the Syndicate allow the Wasp to have enough weapon space to equip a Particle Cannon, the main competitor to the Proton Gun?

**Comment by Nescio0** [comment]
Moved capitalization consistency to #3880, as requested. (Hopefully we won't get a `git` commit conflict.)

**Comment by endless-sky** [comment]
These changes really don't make sense to me. The human interceptors are *meant* to be weak, limited, and inexpensive. Making them capable of carrying more or larger guns would require boosting their price, and thus eroding some of the difference between interceptors and light warships.

**Comment by Pointedstick** [comment]
> The human interceptors are meant to be weak, limited, and inexpensive.

That may describe the Sparrow, but it doesn't fit the the Fury or especially the Hawk. Those latter two ships are clearly larger and heavier and designed to be bomber-type craft.

**Comment by Pointedstick** [comment]
> Although its maximum speed (c. 600 per second) is above average, it is not really exceptional.

Your definition of what's fast may be a bit narrower than is typical. :stuck_out_tongue: 

I would like to see a more focused variant of this go in, but I don't feel that it's merge-worthy right now because changes that make sense are lumped together with changes that are controversial. If you want this to have a chance of getting merged, I think it might make more sense to focus here on the set of non-controversial tweaks, and make your case for other ones elsewhere. No sense in jeopardizing a nice tweak by lumping it together with something contentious. :)
